core/integration/container_test.go

```diff
@@ -502,7 +688,7 @@ func TestContainerMountExec(t *testing.T) {
 			"id": id,
 		}})
 	require.NoError(t, err)
-	require.Equal(t, "hi\n", execRes.Container.From.WithMountedDirectory.Exec.Directory.File.Contents)
+	require.Equal(t, "v18.10.0\ngo version go1.18.2 linux/amd64\n", execRes.Container.From.WithMountedDirectory.Exec.From.Exec.Exec.Stdout.Contents)
```

Comment:
amd64 will fail on arm laptops. Until multiarch is merged then I guess we can just assert on all but that last part of the string? Or something equivalent

Reply:
Ack, great catch! Fixing.